src/theory/bv/bv_solver_bitblast.h

```diff
@@ -89,6 +90,9 @@ class BVSolverBitblast : public BVSolver
    */
   void handleEagerAtom(TNode fact, bool assertFact);
 
+  /** Convert drat proof to proof nodes **/
```

Comment:
Improve the documentation. It is not clear what converting to proof nodes means. Moreover, the return type is a vector of *Node*, not of *ProofNode*. What is necessary is to understand exactly what is the return result of this method.